verification/sparse_copy/CMakeLists.txt

```diff
@@ -1 +1,2 @@
-add_metall_executable(verify_sparse_copy verify_sparse_copy.cpp)
\ No newline at end of file
+add_metall_executable(verify_sparse_copy verify_sparse_copy.cpp)
+add_metall_executable(verify_sparse_copy_syscalls verify_sparse_copy_syscalls.cpp)
```

Comment:
The new verification program is nice. It gives us confidence in the new sparse copy functions.

Apparently, verify_sparse_copy_syscalls.cpp can not be built on MacOS, which is totally okay with me.
Let's enable the target only on Linux, i.e., something like this:

```cmake
if (Linux)
  add_metall_executable(verify_sparse_copy_syscalls verify_sparse_copy_syscalls.cpp)
end()
```